src/code_80280000.c

```diff
@@ -103,6 +104,9 @@ void credits_loop(void) {
         } else {
             func_80280000();
             func_80280038();
+#if VARIABLE_WATCH_DEBUG_LIST
```

Comment:
I'm just wondering if this could be more aptly named like
`DISPLAY_DEBUG_INFO`

Reply:
maybe